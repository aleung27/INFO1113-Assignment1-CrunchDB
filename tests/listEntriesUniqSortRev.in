SET q 1 2 9 3 5 5 10 33 4
LIST ENTRIES
SNAPSHOT
UNIQ q
LIST ENTRIES
SORT q
LIST ENTRIES
REV q
LIST ENTRIES
ROLLBACK 1
LIST ENTRIES
