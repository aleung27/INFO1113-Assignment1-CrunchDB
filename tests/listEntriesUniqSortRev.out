> ok

> q [1 2 9 3 5 5 10 33 4]

> saved as snapshot 1

> ok

> q [1 2 9 3 5 10 33 4]

> ok

> q [1 2 3 4 5 9 10 33]

> ok

> q [33 10 9 5 4 3 2 1]

> ok

> q [1 2 9 3 5 5 10 33 4]

